.github/workflows/node.js.yml

```diff
@@ -10,16 +10,54 @@ on:
         branches: [master]
 
 jobs:
+    add_enviroment_variables:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Add environment variables to env file"
+              run: |
+                  $(pwd)/.github/workflows/env_variables.sh $GITHUB_ENV
+
+    clone_vector_skin:
+        runs-on: ubuntu-latest
+        needs: add_enviroment_variables
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Clone vector skin repository"
+              run: |
+                  git clone "https://gerrit.wikimedia.org/r/mediawiki/skins/Vector" skins/Vector
+
+    mediawiki_docker:
+        runs-on: ubuntu-latest
+        needs: clone_vector_skin
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Add docker-compose.override.yml"
+              run: |
+                  $(pwd)/.github/workflows/docker_compose_override.sh ${{ env.MW_DOCKER_UID }} ${{ env.MW_DOCKER_GID }}
+            - name: "Setup mediawiki-docker"
+              run: |
+                  docker-compose up -d
+                  docker ps -a
+                  docker-compose exec -T mediawiki composer update
+                  docker-compose exec -T mediawiki /bin/bash /docker/install.sh
+            - name: "Add vector skin config to LocalSettings.php file"
+              run: |
+                  sudo bash -c "echo 'wfLoadSkin( 'Vector' );' >> LocalSettings.php"
     test:
         runs-on: ubuntu-latest
+        needs: mediawiki_docker
```

Comment:
Please add a `cat` command for the `.env` and the `LocalSettings.php` file just to make sure that the CI is correctly set up

Reply:
@und3fined-v01d, I will need to run everything in a single job. I added cat but the LocalSettings.php is not being found and the .env seems to be empty.